.travis.yml

```
@@ -9,6 +9,6 @@ script:
 deploy:
   provider: surge
   project: ./build/
-  domain: https://kentico-cloud-sample-app-react-ga.surge.sh
+  domain: https://kontent-sample-app-react.surge.sh
```

Please use -ga suffix:

```suggestion
  domain: https://kontent-sample-app-react-ga.surge.sh
```